tasks/databases.yml

```diff
@@ -43,6 +43,8 @@
   failed_when: hstore_ext_result.rc != 0 and ("already exists, skipping" not in hstore_ext_result.stderr)
   changed_when: hstore_ext_result.rc == 0 and ("already exists, skipping" not in hstore_ext_result.stderr)
   when: item.hstore is defined and item.hstore
+  environment:
+    - PGPORT: "{{ postgresql_port }}"
```

Comment:
Instead of using the `PGPORT` variable, should we just include the `--port={{ postgresql_port }}` parameter on each `psql` call, so that it works similar to `--username` ?

Reply:
yes, i agree on that, it would be cleaner